app/templates/components/ui-table/cell/cell-simple-date.hbs

```diff
@@ -1,6 +1,6 @@
-{{#if record.submittedAt}}
+{{#if (get record column.propertyName)}}
   <span>
-    {{moment-format record.submittedAt 'MMMM DD, YYYY - HH:mm A'}}
+    {{moment-format (get record column.propertyName) column.dateFormat}}
```

Comment:
Make the dateFormat optional. For most cases, `MMMM DD, YYYY - HH:mm A` would be enough. So keep it as default.